packages/cephalon/src/actions/forward-attachments.scope.ts

```
@@ -2,7 +2,7 @@ import { makePolicy, type PolicyChecker } from "@promethean/security";
 import { checkPermission } from "@promethean/legacy";
 
 import type { Bot } from "../bot.js";
-import { makeLogger, type Logger } from "../factories/logger.js";
+import { createLogger, type Logger } from "@promethean/utils/logger.js";
```

ESLint

[error] `@promethean/utils/logger.js` import should occur before type import of `../bot.js`

(import/order)